instructor/common.py

```diff
@@ -211,7 +212,10 @@ def course_modify(request):
             user = User.objects.get(username=request.user)
             for obj in formset.deleted_objects:
                 if obj.owner == user and obj != assignment.course:
```

Comment:
If the course is selected I don't see the error message when attempting to delete it. Can you add an error message for this case?